www/pages/docs/build.md

```diff
@@ -0,0 +1,36 @@
+---
+label: 'build'
+menu: side
+title: 'Build'
+index: 4
+linkheadings: 3
+---
+
+# Build Configurations
+
+A number of core build configuration files can be overridden by creating each file within the root path of your project.
+
+
+### PostCSS
+
+To override the default **postcss.config.js** with your own configuration, create a new postcss.config.js file within your project's root directory.
+
+By default, [this is the postcss.config.js](https://github.com/ProjectEvergreen/greenwood/blob/master/packages/cli/src/config/postcss.config.js) being used.
+
+
+### Babel
+
+To override the default **babel.config.js** with your own configuration, create a new babel.config.js file within your project root directory.
+
+> Note: If you use your own **babel.config.js** you need to include your own [.browserslist](#browserslist) file within the project's root directory
```

Comment:
Actually, this isn't entirely the case is it?
> @babel/preset-env also has an option to set the config path by default we have it set to use the current directory of the babel.config.js

So in theory, they could still use our _browserslistrc_ by passing the path _./node_modules/@greenwood/cli/src/config/.browserslistrc_.  Not sure if that is worth mentioning?

Reply:
Well just in case I added it. Can only use ENV variables or modify package.json to set path to browserlist elsewhere for postcss